packages/tools/pie-icons-webc/tests/icons-react.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/experimental-ct-react';
+import { IconAlcohol } from '../icons/react/IconAlcohol';
+
+test.use({ viewport: { width: 500, height: 500 } });
+
+test('should render', async ({ page, mount }) => {
+    await mount(<IconAlcohol />);
+    expect(await page.screenshot()).toMatchSnapshot('IconAlcohol.png');
```

Comment:
myself and @siggerzz agreed for now that we will add two snapshot tests that verify the component renders with and without props. We will create a larger ticket to develop a more robust suite of tests 👍